packages/data-models/src/constants.js

```diff
@@ -12,6 +14,20 @@ export const LANG_GEEZ = Symbol('ge\'ez')
 export const LANG_CHINESE = Symbol('chinese')
 export const LANG_SYRIAC = Symbol('syriac')
 
+/**
+ * Constants that define a macrolanguage.
+ *
+ * @enum {string} */
+export const Lang = {
+  LATIN: new Language('lat'),
+  GREEK: new Language('grc'),
+  ARABIC: new Language('ara'),
+  PERSIAN: new Language('per'),
+  GEEZ: new Language('gez'),
+  CHINESE: new Language('zho'),
+  SYRIAC: new Language('syr')
+}
+
```

Comment:
I'm not sure myself. I was not able to find any information about approach like this beings used, but I also did not find any info about whether using it is bad. But it's probably better to rethink the design because of the circular reference problem described above.